src/coach/app.py

```diff
@@ -102,19 +98,25 @@ def create_wod():
         
         return jsonify(response.model_dump()), 200
 
-        
     except requests.RequestException as e:
         return jsonify({"error": f"Failed to fetch user history: {str(e)}"}), 500
 
+@app.route("/wod/<string:user_email>", methods=["GET"])
+def get_wod(user_email):
+    """
+    Return the generated WOD for a user.
+    """
+    wod = get_wod_for_user(user_email)
```

Comment:
If I understand correctly get wod will return a new WOD each time instread of storing the generated wod?